dpnp/dpnp_iface_indexing.py

```diff
@@ -1511,15 +1511,24 @@ def take_along_axis(a, indices, axis):
         Indices to take along each 1d slice of `a`. This must match the
         dimension of the input array, but dimensions ``Ni`` and ``Nj``
         only need to broadcast against `a`.
-    axis : int
+    axis : {None, int}
```

Comment:
Could you add a colon for `axis` in `def put` docstrings  like `axis : {None, int}, optional`

Reply:
I don't feel I got the comment.
The `axis` here is not optional, so we can to change to `axis : {None, int}, optional`.
It wasn't implemented as an optional keyword, because in numpy it is not the case.

Reply:
Do you mean to make `axis` keyword as optional one?
It was not implemented in that way, because it is not the case for numpy also. I'd propose to keep the behavior aligned.

Reply:
I meant that `put` documentation for axis parameter there is no colon `:`
This does not apply to your changes here, just noticed it

Reply:
I see, updated.